packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -162,9 +162,9 @@ class WidgetTester extends WidgetController implements HitTestDispatcher {
     Duration duration,
     EnginePhase phase = EnginePhase.sendSemanticsTree
   ]) {
-    return TestAsyncUtils.guard(() {
+    return TestAsyncUtils.guard(() async {
```

Comment:
do you still need this change? Looking at the code more it seems like it's intentionally not async.